### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jonthegeek**
Video for cohort 4, chapter 2 (still processing but should be there in 3 minutes).

Hmm, I forgot that I added it in the wrong place until I tried to submit this. May delete this and try again if the UI doesn't easily let me edit...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ This repo contains 3 main folders:
 *Cohort 4 (started 2020-10-05) Mondays, 17:00 ET*
 
 - 2020-10-05 Chapter 1 - Introduction: Emily Wissel
```
Comment (tanho63) [inline README.md]: We tend to organise these as only "upcoming" meetings - so if those meetings have already happened they should be removed from the list!

Comment (tanho63) [reply README.md]: lmao i see you on this already

**Comment by jonthegeek** [comment]
Ok, I really want to just merge at this point so I don't get lost in the changes :D

**Review by tanho63 — APPROVED** [review]
LGTM

**Pull request merged**